sql/src/main/java/io/crate/rest/action/RestResultSetReceiver.java

```diff
@@ -116,7 +117,7 @@ public void fail(@Nonnull Throwable t) {
             channel.sendResponse(new CrateThrowableRestResponse(channel,
                 createSQLActionException(t, exceptionAuthorizedValidator)));
         } catch (Throwable e) {
-            LOGGER.error("failed to send failure response", e);
+            LOGGER.error("Failed to send failed request response.", e, t);
```

Comment:
I would phrase it: Failed to send error response for failed request. or keep it as it was.

Reply:
Better 👍